src/CalcManager/CEngine/calc.cpp

```diff
@@ -1,9 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include "pch.h"
+#include <assert.h>
```

Comment:
```suggestion
#include <cassert>
```

Reply:
There are at least two other places in this PR with same change

Reply:
Thanks!